pkg/reconciler/taskrun/resources/apply_test.go

```diff
@@ -1103,19 +1086,6 @@ func TestApplyResources(t *testing.T) {
 			spec.Steps[8].Image = "/foo/builtImage"
 			spec.Steps[9].Image = "/workspace/foo/builtImage"
 		}),
-	}, {
-		name: "input resource specified",
```

Comment:
I think this is OK to delete fully because despite the test name it's really a test for the git pipelineresource mutations it looks like